examples/fizzybuzzy.js

```diff
@@ -8,7 +8,8 @@
  * In this modified version, we take in an arbitrary list of numbers as input
  * and return a list of string / number results instead of printing
  */
-function fizzBuzzList(inputs) {
+const inputs = [...Array(20).keys()];
+
```

Comment:
the indentation on this file is cursed, did you mean to do that?

Reply:
nope let me fix it LOL

Reply:
I'm not sure we'll actually be able to infer the type of this, maybe just hardcoding a list of ints would be better.

Reply:
fixed @jlu5